home/models.py

```diff
@@ -784,22 +791,28 @@ class ManifestSettings(models.Model):
             ('BROWSER', 'browser')
         ],
         verbose_name=_("Browser UI"),
-        help_text=_("Provide browser UI"),
+        help_text=_(
+            "Determines the preferred display mode for the website. The possible values are "
+            "'fullscreen' 'standalone' 'minimal-ui' 'browser'"
```

Comment:
Could we elaborate on what each of these options do (This could also be via a link to some external site explaining it), and what the most sensible choice would be for our setting?